pulsar-testclient/src/main/java/org/apache/pulsar/testclient/PerformanceProducer.java

```diff
@@ -479,6 +479,19 @@ public static void main(String[] args) throws Exception {
 
             oldTime = now;
         }
+        executorShutdownNow();
+    }
+
+    static void executorShutdownNow() {
```

Comment:
Just inline or `private static`?

Reply:
> Just inline or `private static`?

private static, I have updated,thanks @tisonkun

Reply:
> Can you elaborate a bit why we need to explicitly shutdown the executor? If the process exits, all tasks just halted.

I think it's more elegant to actively close, control the shutdown wait time,
If the main thread exits, the thread may run incomplete @tisonkun